motoman_gp25_support/package.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>motoman_gp25_support</name>
+  <version>0.3.5</version>
+  <description>
+    <p>ROS-Industrial support for the Motoman gp25 (and variants).</p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Motoman gp25 manipulators.
+    </p>
+    <p>
+      <b>Specifications</b>
+    </p>
+    <ul>
+      <li>gp25 - Default</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information 
+      found in the online 
+      https://www.motoman.com/hubfs/Robots/gp25.pdf
+      All urdfs are based on the default motion and joint velocity limits, 
+      unless noted otherwise.
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author>Eric Marcil</author>
+  <maintainer email="shaun.edwards@gmail.com">Shaun Edwards</maintainer>
+  <maintainer email="g.a.vanderhoorn@tudelft.nl">G.A. vd. Hoorn (TU Delft Robotics Institute)</maintainer>
+  <license>BSD</license>
+
+  <url type="website">http://wiki.ros.org/motoman_gp25_support</url>
```

Comment:
This may be in preparation for build farm release, but technically this website does not exist.

Reply:
@gavanderhoorn:  Will this be created or should it be removed?

Reply:
Yes, we'll create them.